dali/python/nvidia/dali/_utils/eager_utils.py

```diff
@@ -139,3 +198,444 @@ def _slice_tensorlist(data, size):
     """ Constructs TensorList consisting of ``size`` first elements of ``data``. """
 
     return type(data)(list(data)[:size], layout=data.layout())
+
+
+def _arithm_op(name, *inputs):
```

Comment:
Can I get a docstring here that explains that it is used to implementing eager operators that are injected to TensorLists or exposed in .eager.math by using the already exposed stateless eager `arithmetic_generic_op`

Reply:
done